src/components/task-list/index.md.njk

```diff
@@ -0,0 +1,145 @@
+---
+title: Task list
+description: The task list component displays all the tasks a user needs to do, and allows users to easily identify which ones are done and which they still need to do.
+section: Components
+aliases:
+backlogIssueId: 72
```

Comment:
@stevenjmesser Shall we add a separate GitHub issue for the component?

This one is tagged "pattern" (issue ID 72)

* https://github.com/alphagov/govuk-design-system-backlog/issues/72

But there aren't many tagged both "pattern" and "component"

https://github.com/alphagov/govuk-design-system-backlog/issues?q=is%3Aopen+is%3Aissue+label%3Acomponent+label%3Apattern

Reply:
Tag it as both, I think.

Reply:
@frankieroberto @stevenjmesser Done ✅